### ISSUE TITLE
Unicode strings with encoding declaration are not supported

### ISSUE THREAD

**Issue by retr0h** [twill-tools/twill#5]
Ran into the following issue when trying to submit to an endpoint, when endpoint missing default encoding.

```
  File "/Users/jodewey/git/twill/twill/commands.py", line 348, in submit
    browser.submit(submit_button)
  File "/Users/jodewey/git/twill/twill/browser.py", line 382, in submit
    self.result = ResultWrapper(r)
  File "/Users/jodewey/git/twill/twill/utils.py", line 52, in __init__
    self.tree = html.fromstring(self.text)
  File "/Users/jodewey/tests/.venv/lib/python3.7/site-packages/lxml/html/__init__.py", line 875, in fromstring
    doc = document_fromstring(html, parser=parser, base_url=base_url, **kw)
  File "/Users/jodewey/tests/.venv/lib/python3.7/site-packages/lxml/html/__init__.py", line 761, in document_fromstring
    value = etree.fromstring(html, parser, **kw)
  File "src/lxml/etree.pyx", line 3237, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1871, in lxml.etree._parseMemoryDocument
ValueError: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
```

**Comment by Cito** [comment]
Thanks for reporting. Any idea how we can create a test for this? I'd like to make sure that this works with both Python 2 and 3.

**Comment by retr0h** [comment]
Good question.  I can't submit the URL we originally found the issue on, due to it's sensitivity, and figured you would be asking this very question :)

**Comment by Cito** [comment]
Ok, I could reproduce it now using your error message, and added a test. Can you check whether my fix work for you, too? I don't want to encode() because that might break with non-ascii content. Maybe always using content() instead of text() would work, too.

**Comment by retr0h** [comment]
hi @Cito your fix appears to correct that problem.  Thank you!

**Comment by retr0h** [comment]
When possible could you cut a release to pypi?

**Comment by Cito** [comment]
This has just been released in version 2.0.1.
